webpack.config.js

```diff
@@ -45,7 +45,7 @@ const config = {
       },
       {
         test: /(\.tsx?$)/,
-        exclude: /node_modules/,
+        exclude: /node_modules\/(?!(@artsy\/reaction-force)).*/,
```

Comment:
This excludes any modules in `node_modules/` but not `mode_modules/@artsy/reaction-force` so the TS files in `reaction` will be transpiled properly.

Reply:
@yuki24 - Your explanation is correct! (Please keep closely in mind that all of this stuff is in flux right now.)

Reply:
I would not depend on anything from the `src` folder right right now as things could change.

Reply:
I just updated this to `/node_modules\/(?!(@artsy\/reaction-force\/src\/__generated__)).*/,` since the only files that need to be transpiled are auto-generated files whose names follow the pattern `./node_modules/@artsy/reaction-force/src/__generated__/*.graphql.ts`. This should exclude anything in:

 * `node_modules/@artsy/reaction-force/src/`
 * `node_modules/@artsy/reaction-force/dist/`

@damassi What do you think?

Reply:
Seems good to me for now 👍